source/includes/_webhooks.md.erb

```diff
@@ -1,24 +1,59 @@
+To begin using webhooks, as an admin homepage and choose Settings > Integrations > Webhooks.  To begin sending data, you'll need to add a "New Webhook Endpoint." Each Webhook Endpoint URL you specify will receive a stream of all the hooks that occur within your ControlShift account.
 
-If you need additional information about webhooks or how to use them, please send us a support email [support@controlshiftlabs.com](mailto:support@controlshiftlabs.com). We'd also love stories about how customers are using webhooks.
+If you need additional information about webhooks or how to use them, please send us a support email [support@controlshiftlabs.com](mailto:support@controlshiftlabs.com).
 
 ### Retries
 
 For webhook notifications successfully processed the response should have an HTTP 2XX status code. Responses returning a different status code will be marked as failed and automatically retried up to 9 times with an exponential backoff (e.g.: after 15s, 30s, 90s, etc.).
 
 If after 9 retries the notification cannot be delivered, the webhook endpoint will be automatically disabled and will need to be re-enabled from the admin page to start receiving webhook notifications again.
 
+### Security
+
+Securing access to your endpoints where webhooks are sent is delivered. We recommend taking a variety of steps.
+
+* Using HTTPS with modern cypher suite.
+* Restricting access by IP address
+* Using basic HTTP authentication or an API token/secret in the URL.
+* Verifying the signature hash of the message payload we send to authenticate ControlShift as the source of the webhook.
+
+### Webhook HMAC Authenticity Signing
+
+We send a special HTTP header along with webhook deliveries that allows customers to verify that the body of the
+webhook being delivered originated from within your ControlShift account. We use HMAC and SHA 256 along with a
+secret generated for each endpoint
+
+You can retrieve the secret or generate a new one via Settings > Integrations > Webhooks within your ControlShift platform admin.
+
+Header | Value
+---------- | -------
+`x-controlshift-webhook-signature` | sha256:abc123hjkl6789
+
+The value is the prefix `sha256:` specifying the algorithm in use combined with the [HMAC digest](https://en.wikipedia.org/wiki/HMAC) of the webhook HTTP
+request body signed with the signing secret specified for your endpoint in the ControlShift admin tools.
+
+To verify the webhooks you receive are authentic, users should calculate the same digest value and compare it to the digest
+passed in the `x-controlshift-webhook-signature` header. These values should always be the same.
+
+
 ### IP Addresses
 
-Below you can find a list of the IP addresses that webhook deliveries will originate from, in case you want to whitelist them:
+Webhooks will be delivered from a stable set of the IP addresses tied to your region that will not change. It is a good idea to safe-list these addresses and block webhooks originating from unknown IP addresses.
```

Comment:
Is the "tied to your region" part here future-proofing for when we add AWS Frankfurt (and then the ones below will just be the us-east IPs)?